keras_cv/bounding_box/utils.py

```diff
@@ -87,3 +87,104 @@ def _format_outputs(boxes, squeeze):
     if squeeze:
         return tf.squeeze(boxes, axis=0)
     return boxes
+
+
+def pad_with_sentinels(bounding_boxes, default_value=-1):
+    """Pads the given bounding box tensor with -1s.
+
+    This is done to convert RaggedTensors to be converted into
+    standard Dense tensors, which have better performance and
+    compatibility within the TensorFlow ecosystem.
+
+    Args:
+        bounding_boxes: a ragged tensor of bounding boxes.
+        Can be batched or unbatched.
+
+    Returns:
+        a Tensor containing the -1 padded bounding boxes.
+    """
+    return bounding_boxes.to_tensor(default_value)
```

Comment:
what I mean't before was:

```
    return bounding_boxes.to_tensor(default_value=-1)
```

to_tensor has a named argument `default_value`.  

Instead of default_value, lets call our argument `sentinel_value=-1`.

```
def pad_with_sentinels(bounding_boxes, sentinel_value=-1):
    return bounding_boxes.to_tensor(default_value=sentinel_value)
```